symfony/symfony
# Use short array syntax

| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Symfony is one of last projects using traditional array syntax. After dropping support for 2.8, this is best opportunity to switch to short syntax usage. If we do it later, there would be even more places where it will be needed to change and even more conflicts.

This patch is not just running php-cs-fixer. This is a subset of hunk changes which are conflict free!

Great care was taken to:
- Make this commit **mergeable  to master, 4.1 and 4.2 without conflicts** (removed changes which cause conflicts)
- Make sure there are no syntax errors (php lint)
- Make sure phpunit passes (manually skipped dumped fixtures)
